### ISSUE TITLE
Problem with SMS reception after deleted SMS in memory

### ISSUE THREAD

**Issue by toupetlu** [Xinyuan-LilyGO/LilyGO-T-A76XX#184]
Hello

I begin with the use of A7670E R2 not GPS and I use only ATdebug example

I received SMS and the AT Monitor displayed : + SMS FULL.

So I deleted memory with the AT command : AT+CMGD=,1. Since this command, my module didn't receive message. (Phone call are still received)

How to correct this?

Thank you

Ludovic

**Comment by lewisxhe** [comment]
You can try to send a text message to see if it can be sent normally.

**Comment by toupetlu** [comment]
When I send a text message : "TEST" for example (test realized with 2 differents emmetor phone), nothing happend in the AT serial monitor and the AT command to ask list (AT+CMGL=ALL) displays no new received message.

**Comment by lewisxhe** [comment]
When sending a message using AT+CMGS, the modem does not respond with "OK"? No response at all?

**Comment by toupetlu** [comment]
The modem respond OK and i did this following example : I wrote the AT command : the message is sent and I received it on my personnal phone

AT+CMGS="336.......57" 
>HELLO LUDO
<ctrl-Z>

**Comment by toupetlu** [comment]
I re-try this evening. I resent a text SMS to my A7670e module and I got a AT response at these new message and the all other message I send yesterday (the same number of answer as the same number of sms I sent yesterday)... so, surprising

**Comment by lewisxhe** [comment]
So, now you can only send but not receive text messages, right?

**Comment by toupetlu** [comment]
No, now I receive text messages (SMS) normally. (but it is surprising that I received all the text messages (I sent the day before) at the same time)

**Comment by lewisxhe** [comment]
Well... surprising...

**Comment by domilui** [comment]
hello, I have the same problem...

**Comment by domilui** [comment]
by resetting the A76xx, the SMS are unlocked...

**Comment by domilui** [comment]
it seems that unsubscribing and re-registering on the network unblocks the situation. but unfortunately not all the time. 
so I don't know what to do either...

**Comment by lewisxhe** [comment]
@domilui You can try upgrading to the latest firmware

**Comment by domilui** [comment]
thanks!  I don't know how to reinstall the firmware... I'll look at your example to do it and will come back to you if I can't do it
